build-system/eslint-rules/html-template.js

```diff
@@ -28,7 +28,23 @@ const {
  *   TaggedTemplateExpression: {Function(node: CompilerNode): void}
  * }}
  */
-module.exports = function (context) {
+function create(context) {
+  /**
+   * Tags used in <svg> elements can be safely autofixed from an invalid void
+   * tag like <path /> into its full closed form <path></path>.
+   */
+  const svgAutoFixableVoidTags = new Set([
+    'circle',
+    'ellipse',
+    'line',
+    'path',
```

Comment:
question, is `path` for example a void tag? or would it be what is considered as "foreign elements" per https://html.spec.whatwg.org/multipage/syntax.html#foreign-elements

Reply:
It's not a valid void tag, so I may have named the set inappropriately. The concept here is that they can be autofixed when they're self-closing.

Reply:
yep, just clarifying the terminology since im new to the terms

Reply:
What's clear to me is that they're self-closing, so I renamed it to that.